src/seedu/addressbook/data/person/Person.java

```diff
@@ -87,4 +87,14 @@ public String toString() {
         return getAsTextShowAll();
     }
 
+    /**
+     * Returns a concatenated version of the printable strings of each object.
+     */
+    String getPrintableString(Printable... printables) {
```

Comment:
It is also ok to put this method into another class, for example `Formatter`, to follow the SRP.